ci.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python2
```

Comment:
Hmm how does python get referenced in mac then? I don't own a mac so I can't implement that part well. At best, I can maybe detect the OS type and use python if it's OS and python2 if Linux.